### PR TITLE
add bio

### PR DESCRIPTION
**Pull request by Whimby**

### CODE REVIEW

**Review by justin-bushh — CHANGES_REQUESTED** [review]
Formatting

**Review by james-bradshaw-coding — CHANGES_REQUESTED** [review]
failed linter on my machine

plz fix

File: bio/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ves: earth
```
Comment (justin-bushh) [inline bio/README.md:1]: This could maybe be named something more clear.

Comment (Whimby) [reply bio/README.md:1]: copypaste issue i swear

**Review by justin-bushh — APPROVED** [review]

File: bio/README.md (hunk 1 above)
Comment (justin-bushh) [reply bio/README.md:1]: resolve

**Comment by justin-bushh** [comment]
LgTm

**Pull request merged**